File: ui/composite/TimelineToolBar.py
__author__ = 'toramisu'
from ui import *


class TimelineToolBar(QWidget):
    def __init__(self, parent):
        super(TimelineToolBar, self).__init__(parent)
        hbox = QHBoxLayout(self)
        self.startFrameButton = QPushButton()
        hbox.addWidget(self.startFrameButton)
        self.startFrameButton.setText('startFrame')
        self.startFrameLabel = QLineEdit()
        self.startFrameLabel.setText('1')
        hbox.addWidget(self.startFrameLabel)
        hbox.setSpacing(0)
        hbox.setContentsMargins(0, 0, 0, 0)
        self.resize(TIMELINE_TRACK_PANEL_DEF_WIDTH, 15)
        pass

File: view/timeline/TimelineToolBar.cpp
//
// Created by manoyuria on 2015/6/19.
//

#include "TimelineToolBar.h"

TimelineToolBar::TimelineToolBar(QWidget *parent) : QWidget(parent) {
    QHBoxLayout *hbox = new QHBoxLayout(this);
    hbox->setSpacing(0);
    hbox->setContentsMargins(0, 0, 0, 0);

    QPushButton *startFrameButton = new QPushButton();
    hbox->addWidget(startFrameButton);
    startFrameButton->setText("startFrame");

    resize(TIMELINE_TRACK_PANEL_DEF_WIDTH, 15);
//    self.startFrameLabel = QLineEdit()
//    self.startFrameLabel.setText('1')
//    hbox.addWidget(self.startFrameLabel)
}
